docs/tutorials/smart-contract/jsligo.mdx

```diff
@@ -204,6 +204,12 @@ namespace Counter {
 
 Before you can deploy the contract to Tezos, you must compile it to Michelson, the base language of Tezos contracts.
 
+1. Set the compiler to compile the namespace in your code:
```

Comment:
I would say : 
"to target the namespace to compile in your code

Reply:
That's better, thank you. Updated/